en/components/date_range_picker.md

```diff
@@ -0,0 +1,428 @@
+$color-focused: igx-color($custom-palette, "secondary", 500);
+
+// DATE-RANGE
+$custom-date-range-theme: igx-date-range-picker-theme(
+    $label-color: $color-focused
+);
+
+// INPUT GROUP
+$custom-input-group-theme: igx-input-group-theme(
+  $palette: $custom-palette,
+  $filled-text-color: $text-color,
+  $idle-text-color: $text-color,
+  $focused-text-color: $color-focused,
+  $hover-bottom-line-color: $color-focused,
+  $idle-bottom-line-color: $purple
+);
+
+// CALENDAR
+$roundness: 0.5;
+$custom-calendar-theme: igx-calendar-theme(
+    $palette: $custom-palette,
+    $date-current-text-color: $today-text,
+    $border-radius: $roundness,
+    $date-border-radius: $roundness
+);
+```
+
+#### Using CSS variables
+The last step is to pass the custom themes:
+
+```scss
+    @include igx-css-vars($custom-date-range-theme);
+    @include igx-css-vars($custom-input-group-theme);
+    @include igx-css-vars($custom-calendar-theme);
+```
+
+#### Using Theme Overrides
+
+In order to style components for older browsers, like Internet Explorer 11, we have to use a different approach, since it doesn't support CSS variables. 
+
+If the component is using the [`Emulated`](themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`. To prevent the custom theme to leak into other components, be sure to include the `:host` selector before `::ng-deep`:
+
+ ```scss
+:host {
+    ::ng-deep {
+        @include igx-date-range-picker($custom-date-range-theme);
+        @include igx-input-group($custom-input-group-theme);
+        @include igx-calendar($custom-calendar-theme);
+    }
+}
+```
+
+#### Scoping Styles
+Regarding scoping styles, you should refer to both styling sections [Overlay Scoped Component Styles](overlay_styling.md#scoped-component-styles) and [Input Group Scoping Styles](input_group.md#scoping-styles) as they provide more insights.
+
+<div class="sample-container loading" style="height:600px">
+    <iframe id="styled-daterangepicker-iframe" src='{environment:demosBaseUrl}/scheduling/styled-daterangepicker' width="100%" height="100%" 
+        seamless frameBorder="0" class="lazyload no-theming"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="styled-daterangepicker-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+### Application Demo
+The demo below defines form for flight tickets that uses `igxDateRangePicker` component. The dates should be selected otherwise [`IgxHint`]({environment:angularApiUrl}/classes/igxhintdirective.html) is used to display validation error. The selection of the dates is restricted using [`minValue`]({environment:angularApiUrl}/classes/igxdaterangepickercomponent.html#minvalue) and [`maxValue`]({environment:angularApiUrl}/classes/igxdaterangepickercomponent.html#maxvalue) of the `IgxDateRangePickerComponent`
```

Comment:
```suggestion
The demo below defines a form for flight tickets that uses `igxDateRangePicker` component. The dates should be selected otherwise an [`IgxHint`]({environment:angularApiUrl}/classes/igxhintdirective.html) is used to display a validation error. The selection of the dates is restricted using [`minValue`]({environment:angularApiUrl}/classes/igxdaterangepickercomponent.html#minvalue) and [`maxValue`]({environment:angularApiUrl}/classes/igxdaterangepickercomponent.html#maxvalue) of the `IgxDateRangePickerComponent`
```